src/plugins/plot/src/chart/MCTChartController.js

```diff
@@ -152,7 +150,7 @@ function (
     MCTChartController.prototype.destroy = function () {
         this.isDestroyed = true;
         this.stopListening();
-        _.invoke(this.lines, 'destroy');
+        this.lines.map(line => line.destroy());
```

Comment:
we should use this.lines.forEach here, map should be used in cases where we expect a new array returned

Reply:
done